pages/account/[[...index]].tsx

```diff
@@ -149,6 +156,35 @@ export default function MyAccount({
           </Text>
         ) : (
           <>
+            {fines.total > 0 && (
```

Comment:
Is the below notification component big enough to pull out?

Reply:
Doesn't seem substantial enough to warrant it's own component in my opinion, what do others think?

Reply:
Agreed, this is okay. Also, this will get updated with the new `Banner` component which should be released by the end of March: https://nypl-design-system-git-dsd-1691-banner-nypl.vercel.app/?path=%2Fdocs%2Fcomponents-feedback-banner--docs

Reply:
I think it can come out! especially because there's that big chunk of text, and it doesn't require a ton of props. It's relevant to the parent index page that there could be a notification displayed, but not necessarily what is rendered as a part of that notification.